src/editor/Model/QuickOpen.re

```diff
@@ -0,0 +1,24 @@
+open Oni_Core.Types;
+open UiMenu;
+
+let content = (effects: Effects.t) =>
+  effects.getCurrentDir()
```

Comment:
It looks like right now we only pull in the top-level directory - one enhancement we'll need down the road is to also filter through subdirectories. Delegating that to a utility like `ripgrep` might be helpful.